File: codes/python/Code_2_04_2.py
# 入力
N, X, Y = map(int, input().split())

# 答えを求める
cnt = 0
for i in range(1, N + 1):
	# mod の計算は 2.2 節参照
	if i % X == 0 or i % Y == 0:
		cnt += 1

# 出力
print(cnt)

File: codes/c/Code_2_04_2.c
#include <stdio.h>

int main() {
	// 入力
	int N, X, Y;
	scanf("%d%d%d", &N, &X, &Y);
	
	// 答えを求める
	int cnt = 0;
	int i;
	for (i = 1; i <= N; i++) {
		// mod の計算は 2.2 節参照
		if (i % X == 0 || i % Y == 0) {
			cnt += 1;
		}
	}
	
	// 出力
	printf("%d\n", cnt);
	
	return 0;
}
